.golangci.yaml

```diff
@@ -1,94 +1,96 @@
 linters:
-  enable-all: true
-  disable:
-    - contextcheck  # too many false positives
-    - depguard      # checks if package imports are whitelisted
```

Comment:
Please revert all these whitespace changes in this file

Reply:
There is still a huge changeset here. We should keep the two spaces indentation on these yamls